CONTRIBUTING.md

```diff
@@ -80,3 +80,70 @@ command:
 
 After running, you can view the Storybook at <http://localhost:6006/> in your
 browser.
+
+## Multi-tenancy
```

Comment:
Could use feedback on these instructions, particularly if the contributing guide is the best place to put them

Reply:
Yeah, that's a good question. I think this location makes more sense than e.g. `docs/tutorial.md` or something. We can ask SFDO, but I think this is probably fine.